Source/WebKit/Configurations/WebKit.xcconfig

```diff
@@ -171,10 +171,14 @@ WK_INSTALL_COORDINATION_LDFLAGS = $(WK_INSTALL_COORDINATION_LDFLAGS_$(WK_PLATFOR
 WK_INSTALL_COORDINATION_LDFLAGS_iphoneos = -framework InstallCoordination;
 WK_INSTALL_COORDINATION_LDFLAGS_iphonesimulator = $(WK_INSTALL_COORDINATION_LDFLAGS_iphoneos)
 
+WK_FRONTBOARDSERVICES_LDFLAGS = $(WK_FRONTBOARDSERVICES_LDFLAGS_$(WK_PLATFORM_NAME));
```

Comment:
You might need a tbd in `WebKitLibraries/WebKitPrivateFrameworkStubs/iOS/16` for this to build on the bots.